packages/client/src/Loc.ts

```diff
@@ -406,21 +407,24 @@ export class LocsState extends State {
         if (params.locType === "Identity") {
             await client.openIdentityLoc({
                 ...params,
-                locId
-            });
+                locId,
+                autoPublish: false,
```

Comment:
I thought that in this case, items are directly published, is that done elsewhere?

Reply:
In this case, the value of `autoPublish` is anyway ignored because of the other flag `requirePreOpen` is false. So there is no need. But I agree it's a bit misleading, I'll rewrite this part to make it more self-explanatory.